RHINENG-8806 update README.md file (#163)
RedHatInsights/config-manager: README.md

Config Manager handles the following actions:

> - Retrieves the new configuration (enabled/disabled) of various services supported by Service Enablement 
> - Compares it with the current configuration of the services from the profile table for that org ID and updates the configuration of the services.

Reviewer: ```suggestion
- Receives the new configuration (enabled/disabled) of various services supported by Service Enablement 
- Retrieves the current configuration of the services from the profile record belongs to given org ID and updates the configuration of the services after comparing new vs current configurations.
```

- Maintains a history of configuration changes
- Ensures that newly connected hosts are kept up to date with the latest configuration
- Updates the host's system profile in Inventory with the latest "rhc_config_state" ID

Config Manager has two mechanisms to update a host:

> - Via a change in the Service Enablement dashboard - It will update all the hosts under that org ID.

Reviewer: ```suggestion
- Via a change in the Service Enablement dashboard - updates all the hosts belong to a given org ID.
```

> - Via a new rhc connection event from Inventory - It will update a single host.

Reviewer: ```suggestion
- Via a new rhc connection event from Inventory - updates a single host.
```
